### ISSUE TITLE
For KVM hypervisor in hv_kvm.py was forcibly added option add-usbdevice=tablet that duplicates the parameter usb_mouse

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#165]
Originally reported of Google Code with ID 109.

```
<b>What steps will reproduce the problem?</b>
1. gnt-instance modify -H usb_device="tablet" kvm-guest1
2. gnt-instance startup kvm-guest1
3. ps ax|grep kvm-guest1 show
...-usbdevice tablet..-usbdevice tablet..
-usbdevice repeated twice in kvm command line
that throw windows 2003 guest to blue screen 

problem in /usr/lib/python2.6/dist-packages/ganeti/hypervisor/hv_kvm.py
551       # Also add a tablet USB device to act as a mouse
552       # This solves various mouse alignment issues
553       kvm_cmd.extend(['-usbdevice', 'tablet'])
this lines forgot about parameter usb_mouse
```
Originally added on 2010-07-07 09:59:46 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2010-08-09 16:26:49 +0000 UTC.

Added Labels: Type-Defect Priority-Medium
Added to Milestone: Release2.1

**Comment by Ganeti-Issues-Migrator** [comment]
```
This has been fixed in devel-2.1 and will be part of the next stable release.

Thanks,

Guido

```
Originally added on 2010-08-18 22:25:52 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:08 +0000 UTC.

Changed State: Released
